### DISCUSSION TITLE
How to config a Gravitino Catalog and a Trino For Querying above multiple Hadoop Systems

### DISCUSSION THREAD

**Discussion by theoryxu** [apache/gravitino#2496] [Q&A]
Hi Fellows

I read the docs and found that: https://datastrato.ai/docs/0.4.0/trino-connector/catalog-hive#hdfs-config-and-permissions

So, how do I configure a Gravitino Catalog and a Trino For querying above multiple Hadoop Systems? 

Is there any advice, or do some docs provide more details?

Thanks

**Comment by diqiu50** [comment]
@theoryxu Thank you for the question.
In Trinno docs [hdfs-username-and-permissions.](https://trino.io/docs/current/connector/hive.html#hdfs-username-and-permissions)
We can solve these problems with the ·Kerberos solution·.. The Gravitino 0.4 version does not support it for now. This issue will be resolved in future versions.

**Comment by justinmclean** [comment]
Would creating multiple catalogs (one per system) and then doing cross catalog queries work?
